packages/gestalt/src/Datapoint/DataPointBaseProps.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable no-empty-pattern */
+/* eslint-disable react/no-unused-prop-types */
+// @flow strict
+
+/**
+ * React Prop Types File for React-DocGen to share props across similar components
```

Comment:
This is a lot of additional complexity to avoid repeating a few types. For now, let's just repeat the types.